report_xlsx/static/src/js/report/qwebactionmanager.js

```diff
@@ -0,0 +1,47 @@
+// © 2017 Creu Blanca
+// License AGPL-3.0 or later (https://www.gnuorg/licenses/agpl.html).
+odoo.define('report_xlsx.report', function(require){
+'use strict';
+
+var ActionManager= require('web.ActionManager');
+var crash_manager = require('web.crash_manager');
+var framework = require('web.framework');
+
+ActionManager.include({
+    ir_actions_report: function (action, options){
```

Comment:
Are we sure about this migration? AFAICT this function was completely refactored out.  There is no way this module should work straight from 11.0 to 12.0 without major changes here.  I would have thought at least intercepting _executeReportAction, and to do cleanly also _triggerDownload and _downloadReport rather than just contain the action in one monolithic function. 

But I wonder if we can't just intercept at _makeReportURLs and set an url there and then intercept again at _triggerDownload and pass to a custom _downloadXLSX function or something and ignore a lot of this.

Reply:
@gdgellatly Thanks for your review, so i will try to intercept `_makeReportURLs` and next functions at first try.